src/main/java/treky/TrekyException.java

```
@@ -0,0 +1,7 @@
+package treky;
+
+public class TrekyException extends Exception {
```

seems like a catch-all exception class, maybe having subclasses of this (e.g. `TrekyInvalidCommandException`, etc.) may be useful?